includes/modules/payment/securesubmit.php

```diff
@@ -219,7 +222,19 @@ public function before_process() {
 
         return false;
     }
-
+    
+    public function setConfig()
+    {
+        $config = new ServicesConfig();
+        $config->secretApiKey = MODULE_PAYMENT_SECURESUBMIT_SECRET_KEY;
+        $env = $config->environment;
+        $config->serviceUrl = ($env != "TEST")?
```

Comment:
`$config->environment` defaults to `TEST`, and as far as I know, it's not updated based on the API key. THis would cause this conditional to always be true.

I just found out that recent changes to the SDK (included in this PR) allow the integrator to only supply the secret API key as it determines the service URL based on the API key. We should be able to remove the code manually changing the service URL as long as you're able to verify this is working as expected on your end.